packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -104,53 +104,52 @@ export function Previewer({
             onStartEdit={(e): void => onStartEdit(e, index)}
           />
         ))}
-        <View className="amplify-fileuploader__footer">
+      </View>
+      <View className={ComponentClassNames.FileUploaderPreviewerFooter}>
+        <View>
           {isLoading && (
             <>
               <Text>Uploading: {percentage}%</Text>
               <Loader
-                className="amplify-fileuploader-loader"
+                className={ComponentClassNames.FileUploaderLoader}
                 variation="linear"
                 percentage={percentage}
+                isPercentageTextHidden
                 isDeterminate
               />
             </>
           )}
-          {!isLoading && !isSuccess && (
-            <>
-              <View>
-                <Button
-                  disabled={
-                    fileStatuses.some((status) => status?.error) ||
-                    isEditingName.some((edit) => edit) ||
-                    remainingFilesLength === 0 ||
-                    maxFilesError
-                  }
-                  size="small"
-                  variation="primary"
-                  onClick={onFileClick}
-                >
-                  {translate('Upload')}
-                  {` ${remainingFilesLength} `}
-                  {translate('files')}
-                </Button>
-              </View>
-              <Button size="small" variation="link" onClick={onClear}>
-                {translate('Clear all')}
-              </Button>
-            </>
-          )}
-          {isSuccess && (
-            <>
-              <Text />
-              <Button size="small" onClick={onClear}>
-                {translate('Done')}
-              </Button>
-            </>
-          )}
         </View>
-      </Flex>
+        {!isLoading && !isSuccess && (
+          <Flex direction="row">
```

Comment:
nit: would consider if it makes sense to use a css in terms of performance coz unlike `size` and `variation`, `direction` will pass as inline style prop.